src/pages-content/pricing-data.js

```diff
@@ -71,7 +71,7 @@ const faqs = [
     {
         q: 'Does session recording impact costs?',
         a:
-            'Session recording generates events which are billed the same as any other event type. You can tightly control where it is used to manage the cost. Using session recording heavily will increase costs significantly.',
+            'Session recording is currently free to use. This might change in the future.',
```

Comment:
Does it usually lead to more events (thus more cost) though?

Reply:
Recordings do lead to more events which generate a cost for us. IIRC it's lower than regular events, but @fuziontech can confirm

Reply:
Nope, we don't charge for session recordings at all.

Reply:
This seems to suggest we include $snapshot events: https://github.com/PostHog/posthog-cloud/blob/1f8888fc043bbd8e945c0fd87383a34c2afcb4d3/multi_tenancy/utils.py#L26-L38

Reply:
Actually, no, because session recording events go to table `session_recording_events`

Reply:
Ahhh. I see, we haven't been sending `$snapshot` to events since a long time ago. Thanks